orthanc/orthanc-raw/plugin/pixl.py

```diff
@@ -44,6 +49,21 @@ def OnHeartBeat(output, uri, **request):  # noqa: ARG001
     output.AnswerBuffer("OK\n", "text/plain")
 
 
+def ReceivedInstanceCallback(receivedDicom: bytes, _: str) -> Any:
+    """Optionally record headers from the received DICOM instance."""
+    if ShouldRecordHeaders():
+        record_dicom_headers(receivedDicom)
+    return orthanc.ReceivedInstanceAction.KEEP_AS_IS, None
+
+
+def ShouldRecordHeaders() -> bool:
```

Comment:
Might be worth keeping to snake case for everything that isn't an orthanc callback